tests/ci/cdk/cdk/accp_github_fuzz_ci_stack.py

```diff
@@ -0,0 +1,131 @@
+            description="Allow all traffic inside security group"
+        )
+
+        efs_subnet_selection = ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE_WITH_EGRESS)
+
+        # Create the EFS to store the corpus and logs. EFS allows new filesystems to burst to 100 MB/s for the first 2
+        # TB of data read/written, after that the rate is limited based on the size of the filesystem. As of late
+        # 2021 our corpus is less than one GB which results in EFS limiting all reads and writes to the minimum 1 MB/s.
+        # To have the fuzzing be able to finish in a reasonable amount of time use the Provisioned capacity option.
+        # For now this uses 100 MB/s which matches the performance used for 2021. Looking at EFS metrics in late 2021
+        # during fuzz runs EFS sees 4-22 MB/s of transfers thus 100 MB/s gives lots of buffer and allows ~4-5 fuzz runs
+        # to start at the same time with no issue.
+        # https://docs.aws.amazon.com/efs/latest/ug/performance.html
+        fuzz_filesystem = efs.FileSystem(
+            scope=self,
+            id="{}-FuzzingEFS".format(id),
+            file_system_name="AWS-LC-Fuzz-Corpus",
+            enable_automatic_backups=True,
+            encrypted=True,
+            security_group=build_security_group,
+            vpc=fuzz_vpc,
+            vpc_subnets=efs_subnet_selection,
+            performance_mode=efs.PerformanceMode.GENERAL_PURPOSE,
+            throughput_mode=efs.ThroughputMode.PROVISIONED,
+            provisioned_throughput_per_second=Size.mebibytes(100),
+        )
+
+        # Define CodeBuild.
+        fuzz_codebuild = codebuild.Project(
+            scope=self,
+            id="FuzzingCodeBuild",
+            project_name=id,
+            source=git_hub_source,
+            role=role,
+            timeout=Duration.minutes(120),
+            environment=codebuild.BuildEnvironment(compute_type=codebuild.ComputeType.LARGE,
+                                                   privileged=True,
+                                                   build_image=codebuild.LinuxBuildImage.STANDARD_4_0),
+            build_spec=BuildSpecLoader.load(spec_file_path),
+            vpc=fuzz_vpc,
+            security_groups=[build_security_group])
+        fuzz_codebuild.enable_batch_builds()
+
+        # CDK raw overrides: https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html#cfn_layer_raw
+        # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#aws-resource-codebuild-project-properties
+        # The EFS identifier needs to match tests/ci/common_fuzz.sh, CodeBuild defines an environment variable named
+        # codebuild_$identifier.
+        # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-codebuild-project-projectfilesystemlocation.html
+        #
+        # TODO: add this to the CDK project above when it supports EfsFileSystemLocation
```

Comment:
note for later PR: does CDK support this now?

Reply:
Will look into it.